org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/DebugAdapter.java

```diff
@@ -11,6 +11,43 @@
 
 package org.eclipse.jdt.ls.debug.adapter;
 
+import java.io.IOException;
```

Comment:
i used the eclipse context menu 'organize imports' to remove unnecessary imports, and seems it reformat the positions